Framework/PythonInterface/plugins/algorithms/LoadWANDSCD.py

```diff
@@ -0,0 +1,169 @@
+from __future__ import (absolute_import, division, print_function)
+from mantid.api import PythonAlgorithm, AlgorithmFactory, PropertyMode, WorkspaceProperty, Progress, MultipleFileProperty, FileAction, mtd
+from mantid.kernel import Direction, Property, IntArrayProperty, StringListValidator
+from mantid.simpleapi import LoadEventNexus, RemoveLogs, DeleteWorkspace, ConvertToMD, Rebin, CreateGroupingWorkspace, GroupDetectors, SetUB
+import numpy as np
+import h5py
+import re
+
+
+class LoadWANDSCD(PythonAlgorithm):
+
+    def category(self):
+        return 'DataHandling\\Nexus'
+
+    def name(self):
+        return 'LoadWANDSCD'
+
+    def summary(self):
+        return 'Load WAND single crystal data into a detector space vs rotation MDHisto'
+
+    def PyInit(self):
+        self.declareProperty(MultipleFileProperty(name="Filename", action=FileAction.OptionalLoad,
+                                                  extensions=[".nxs.h5"]),
+                             "Files to load")
+        self.declareProperty('IPTS', Property.EMPTY_INT, "IPTS number to load from")
```

Comment:
Just a personal opinion, I think folder would be more appropriate. Otherwise you assume the mounting point of the data

Reply:
I think this is easier for users. This will be removed once we get OnCat support in mantid.

Reply:
[`GetIPTS`](http://docs.mantidproject.org/nightly/algorithms/GetIPTS-v1.html) uses `FileFinder` and goes up a couple of directories. This assumes that the data is mounted, but not where. Is it a "good" middle ground?

Reply:
But `FileFinder` still hits the catalog for the file location, unless I'm misunderstanding something? And HB2C is not in Icat, hence I will fix this once OnCat is supported in Mantid.